src/doc/users-guide/install.md

```diff
@@ -5,78 +5,86 @@ eleventyNavigation:
  - FreeBSD (13)
- - Fedora (32)
- - macOS (10.15, 11.2)
+ - Fedora (34, 36)
+ - macOS (10.15, 11.6, 12.4)
 
-## Download
-
-Use the links below to download the appropriate OpenCilk precompiled shell
-archive (`.sh`), tarball (`.tar.gz`), or Docker image.
-
-You can also [build OpenCilk from source](../build-opencilk-from-source), which is the recommended approach for Ubuntu 18.04 and other operating systems not listed above.
-
-### Linux
- 
- - [{{ download.x86.shell }}]({{ download.host }}releases/download/{{ download.release }}/{{ download.x86.shell }})
-   ({{ download.x86.size }})
- - <a id="{{ download.release }} x86" href="{{ download.host }}releases/download/{{ download.release }}/{{ download.x86.binary }}">{{ download.x86.binary }}</a>
-   ({{ download.x86.size }})
-   
 {% alert "info" %}
-***Note for non-macOS Linux systems:*** You need to install GCC (or equivalent) to provide
-necessary system include files and system libraries.
+***Prerequisites:*** The OpenCilk binaries assume that system header files and system libraries
+are already installed on the system.
+- On Linux, we recommend installing GCC (or equivalent) to provide these necessary system files.
+- On macOS, you must install [XCode](https://developer.apple.com/support/xcode/) or
+the [XCode Command Line Tools](https://mac.install.guide/commandlinetools/index.html) to get
+the necessary system files. 
 {% endalert %}
 
-### macOS
+## Methods of installation
 
- - [{{ download.arm.shell }}]({{ download.host }}releases/download/{{ download.release }}/{{ download.arm.shell }})
-   ({{ download.arm.size }})
- - <a id="{{ download.release }} arm" href="{{ download.host }}releases/download/{{ download.release }}/{{ download.arm.binary }}">{{ download.arm.binary }}</a>
-   ({{ download.arm.size }})
+Precompiled binaries for OpenCilk 2.0 can be installed in several ways:
+- Using a [shell archive (`.sh`)](#installing-using-a-shell-archive) built for the target hardware and operating system.
+- Using a [tarball (`.tar.gz`)](#installing-using-a-tarball) built for the target hardware and operating system.
+- Using a [docker image](#docker-image).
 
-{% alert "info" %}
-***Note for macOS users:*** Unless you are using the OpenCilk Docker image,
-you must also install [XCode](https://developer.apple.com/support/xcode/) or
-the [XCode Command Line
-Tools](https://mac.install.guide/commandlinetools/index.html), which provide
-standard system libraries and header files needed by the OpenCilk compiler.
-{% endalert %}
+You can also [build OpenCilk from source](../build-opencilk-from-source),
+which is the recommended approach for Ubuntu 18.04 and other operating systems for
+which precompiled binaries are not available.
 
-### Docker
+## Installing using a shell archive
 
- - <a id="{{ download.release }} docker" href="{{ download.host }}releases/download/{{ download.release }}/{{ download.docker.binary }}">{{ download.docker.binary }}</a>
```

Comment:
The hard-coded `<a>` tag (as opposed to a markdown link) is unfortunately necessary to provide a "hook" for Google Analytics / Google Tag manager to count downloads. I say a little about this in #20. I will take a crack at adding the <a> tags back in below

Reply:
Here is a screenshot showing how Google Tag manager has three different triggers that are for "just links" and are further filtered based on the `id` of the link:
![Screenshot 2022-08-05 at 17-48-13 Google Tag Manager](https://user-images.githubusercontent.com/93284810/183215392-592f0697-6fd8-4823-b90b-32e1cf3d0f3f.png)